backend/medtagger/database/models.py

```diff
@@ -164,6 +165,16 @@ def available_tags(self, new_tags: List['LabelTag']) -> None:
         """Set new Label Tags for this Task."""
         self._tags = new_tags
 
+    @property
+    def number_of_available_scans(self) -> int:
```

Comment:
Please add also a method `get_number_of_labelled_scans`, which will use this query:

```python
def get_number_of_labeled_scans(self, user_id: int) -> int:
    datasets_ids = [dataset.id for dataset in self.datasets]
    number_of_labeled_scans = Scan.query.join(Label).filter(and_(
        Scan.dataset_id.in_(datasets_ids),
        Scan.status == ScanStatus.AVAILABLE,
        Label.owner_id == user_id
    )).count()
```

Also, please make sure to return this number in the REST API endpoint output as well. Dawid needs it on the frontend side to properly render labeling progress.

Reply:
Or use `User` object as an input. Use whatever will be easier for you:

```python
def get_number_of_labeled_scans(self, user: User) -> int:
    ...
```

Reply:
After discussion offline, let's do it in another PR.